### PR TITLE
Minor updates.

### PR DESCRIPTION
**Pull request by david-driscoll**
* Added Container.From
* Do not serialize out the PrivateHandlerId if it's empty
* StringOrMarkupContent implict conversion should be nullable.

cc @NTaylorMullen @333fred

### CODE REVIEW

File: src/Protocol/Models/StringOrMarkupContent.cs [hunk 1]
```diff
@@ -17,9 +17,9 @@ public record StringOrMarkupContent
         public MarkupContent? MarkupContent { get; }
         public bool HasMarkupContent => String == null;
 
-        public static implicit operator StringOrMarkupContent(string value) => new StringOrMarkupContent(value);
+        public static implicit operator StringOrMarkupContent?(string? value) => value is null ? null : new StringOrMarkupContent(value);
```
Comment (333fred) [inline src/Protocol/Models/StringOrMarkupContent.cs:20]: Could have `NotNullIfNotNull`

**Review by 333fred — APPROVED** [review]

**Review by NTaylorMullen — APPROVED** [review]
Excited to have a variant that doesn't require the serialization of the resolver handler ID, /cc @ToddGrun since we'll both probably want to adopt this for perf reasons.

File: src/Protocol/Models/StringOrMarkupContent.cs (hunk 1 above)
Comment (333fred) [reply src/Protocol/Models/StringOrMarkupContent.cs:20]: @david-driscoll doesn't look like this was addressed.

Comment (333fred) [reply src/Protocol/Models/StringOrMarkupContent.cs:20]: @david-driscoll doesn't look like this was addressed.

**Pull request merged**